examples/step-38/doc/results.dox

```diff
@@ -147,7 +147,7 @@ asterisks. It is worth pointing out one other thing here, though: because we
 detach the manifold description from the surface mesh, whenever we use a
 mapping object in the rest of the program, it has no curves boundary
 description to go on any more. Rather, it will have to use the implicit,
-StraightBoundary class that is used on all parts of the boundary not
```

Comment:
While we are here: it is more correct to write 'on all parts of the *domain* not explicitly assigned a different manifold object' since manifolds are a conceptually orthogonal concept to boundary descriptions.

Reply:
Fixed.